core/pom.xml

```diff
@@ -33,7 +33,7 @@
     <url>https://arctic.netease.com</url>
 
     <properties>
-        <rocksdb.version>5.17.2</rocksdb.version>
+        <rocksdb.version>6.20.3-ververica-1.0</rocksdb.version>
```

Comment:
The previous version of rocksdb didn't support enableAutoCompaction API, so I upgraded the rocksdb version, mainly because the flink engine was using it.